+dj/+internal/AutoPopulate.m

```diff
@@ -52,7 +53,22 @@
     end
     
     methods
-        
+        function target = getTarget(self)
```

Comment:
can this be more naturally orrganized as a dependent property?

Reply:
Thanks for the suggestion. I've now implemented this as a getter for the `target_` property. Unfortunately, [Matlab's inheritance rules](https://www.mathworks.com/help/matlab/matlab_oop/modify-superclass-properties.html) don't allow for overrides so there's no simpler way I know of to implement this.